tools/tm-signer-harness/internal/test_harness_test.go

```diff
@@ -73,7 +73,7 @@ const (
 )
 
 func TestRemoteSignerTestHarnessMaxAcceptRetriesReached(t *testing.T) {
-	cfg := makeConfig(t, 1, 2)
+	cfg := makeConfig(t, 200, 2)
```

Comment:
acceptDeadline of 1 ms was not sufficient?

Reply:
reverted. This was temporary and depending on another thing I was working on